go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/bitnami-labs/charts-syncer
 
-go 1.15
+go 1.16
```

Comment:
Any reason no to use the 1.17 for example (the last one)? It's true it's not the scope of this PR, just thinking out loud

Reply:
I tried that version but found a weird error in the tests only happening with go 1.17.
I would try to updated in another PR later